main.py

```
@@ -674,14 +674,19 @@ async def push_processed_response(context, chat_id, target_chat, reply_text, use
-                await context.bot.send_message(chat_id=target_chat, text=chunk)
+                await context.bot.send_message(
+                    chat_id=target_chat,
+                    text=chunk,
+                    reply_to_message_id=target_reply_id,
+                    parse_mode="HTML"
+                )
+            except Exception:
+                await context.bot.send_message(
+                    chat_id=target_chat,
+                    text=chunk
+                )
```

Ruff (0.15.13)

[warning] `try`-`except` within a loop incurs performance overhead

(PERF203)